docs/.vuepress/theme/NavLinks.vue

```diff
@@ -46,21 +46,17 @@ import NavLink from './NavLink.vue'
 export default {
   components: { OutboundLink, NavLink, DropdownLink, CdrAccordion },
   data() {
-    /*
-      Initialize as closed accordions.
-    */
     return {
-      navGroup: [
```

Comment:
😆 every time i saw this i was like "that's not right..." but did not have the will power to fix it